packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -51,7 +51,7 @@ class _DefaultDoctorValidatorsProvider implements DoctorValidatorsProvider {
         _validators.add(androidValidator);
 
       if (iosWorkflow.appliesToHostPlatform)
-        _validators.add(iosValidator);
+        _validators.add(new GroupedValidator(<DoctorValidator>[iosValidator, cocoapodsValidator]));
```

Comment:
Will this affect the presentation of the iOS validation to public users?

Reply:
It shouldn't. Pasted doctor output in a comment below